lib/Slic3r/GUI/PresetEditor.pm

```diff
@@ -939,6 +942,7 @@ sub _update {
     
     my $have_support_material = $config->support_material || $config->raft_layers > 0;
     my $have_support_interface = $config->support_material_interface_layers > 0;
+    my $have_support_pillars = $have_support_material && $config->support_material_pattern == 'pillars';
```

Comment:
Argument "pillars" isn't numeric in numeric eq (==) at /home/lenox/slic3r-release/lib/Slic3r/GUI/PresetEditor.pm line 945.
Argument "rectilinear-grid" isn't numeric in numeric eq (==) at /home/lenox/slic3r-release/lib/Slic3r/GUI/PresetEditor.pm line 945.
Argument "pillars" isn't numeric in numeric eq (==) at /home/lenox/slic3r-release/lib/Slic3r/GUI/PresetEditor.pm line 945.
Argument "pillars" isn't numeric in numeric eq (==) at /home/lenox/slic3r-release/lib/Slic3r/GUI/PresetEditor.pm line 945.

Reply:
Use `eq` instead of `==` and it works.